index.html

```diff
@@ -12,24 +12,30 @@
 </head>
 <body>
   <div class="status-bar">
-    <div class="hour">9:41</div>
-    <div class="wifi"></div>
-    <div class="signal"></div>
-    <div class="battery"></div>
+    <div class="status-hour">9:41</div>
+    <div class="status-bar-space"></div>
+    <div class="status-images">
+      <img src="./Images/status.png">
+    </div>    
   </div>
-  <nav class="navbar">
-     
+  <nav class="navbar">     
     <div class="nav-name">Jhon Doe</div>
-    <div class="nav-menu"></div>      
-        
+    <div class="nav-space"></div>
+    <div class="nav-menu"><img src="./Images/menu.png" height="30px"></div>        
   </nav>
   <div class="container">
     <div class="primary-text">Hey There. I’m Alex</div>
     <div class="supporting-text">I’am a Software Developer</div>
     <div class="description">I can help you build a product , feature or website. 
       Look through some of my work and experience! If you like what you see and have a project you need coded, 
       don’t hesitate to contact me.</div>
-  </div>
-    
+    <div class="icons">
+      <img src="./Images/Icon -GitHub.png">
```

Comment:
- [ ] Let's kindly try to stick to the design as much as possible. The "the margin between the description and the icons should be increased". Kindly change this.

Your design :

![Screenshot from 2022-12-01 09-34-11](https://user-images.githubusercontent.com/80458199/204993727-76dbddc8-e3e2-48b0-9123-7b64f834b83b.png)

The Figma Design : 

![Screenshot from 2022-12-01 09-34-35](https://user-images.githubusercontent.com/80458199/204993862-508ed47e-0c81-4f87-852f-f30462462972.png)